cf-ci-orchestration/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN true \
     && zypper --non-interactive addrepo http://download.opensuse.org/repositories/utilities/openSUSE_Leap_42.3/utilities.repo \
     && zypper --non-interactive --gpg-auto-import-keys refresh \
     && zypper --non-interactive install ruby \
+    python-pip \
```

Comment:
Instead of using pip, we might also consider adding the `aws` cli to our orchestration image via the bundled installation so we aren't installing one-off tools for this CLI. Note that this bundle still *does* use pip, but it uses a pip executable within the bundle that also has all necessary dependencies, so is easy to delete.

In a now-deprecated precursor to our CI, we did this:

```
  cd /tmp
  curl -sS "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
  unzip awscli-bundle.zip
  ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
  rm -rf aws-cli-bundle aws-cli-bundle.zip
```

This installs a recent version of the aws CLI for our use

If we go this route, we can delete line 27 as well

Reply:
I just decided to go with the official way described in Amazon docs

Reply:
This is one of several official ways described in their documentation: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html#install-tool-bundled